build/tutorials/platform/integrate-exchange-with-avalanche.md

```diff
@@ -36,6 +38,12 @@ Interacting with the C-Chain is identical to interacting with [go-ethereum](http
 
 Please note that `personal_` namespace is turned off by default. To turn it on, you need to pass the appropriate command line switch to your node, like in the above config example.
 
+## Integration using Rosetta
+
+[Rosetta](https://www.rosetta-api.org/) is an open-source specification and set of tools that makes integrating with different blockchain networks easier by presenting the same set of APIs for every network.
+
+You can find the Rosetta server implementation for Avalanche C-Chain [here](https://github.com/ava-labs/avalanche-rosetta), all you need to do is install and run the server with proper configuration. You don't necessarily need to even run you own node, and can instead rely on our [Public API](https://docs.avax.network/build/tools/public-api) to provide network access. 
```

Comment:
Nit:

```
avalanche-rosetta is packaged as a single Dockerfile to make it easy to run and manage.
```

More color on this + online/offline mode here: https://www.rosetta-api.org/docs/node_deployment.html

Reply:
I'd add that detailed setup instructions can be found on the repo

Reply:
Added clarification.